### PR TITLE
Pull request for training login changes

### PR DESCRIPTION
**Pull request by Ezmeyers**
Here is the changes made, with sharedloginactivity added to training app

### CODE REVIEW

**Comment by sunbiz** [comment]
The following exception is thrown, after the tracker app is closed and the training app is attempted to start. After which the training app can never be started

> 02-03 04:20:13.799 4789-4789/edu.iupui.soic.biohealth.plhi.mhbs E/AndroidRuntime: 
> 	FATAL EXCEPTION: main
>     Process: edu.iupui.soic.biohealth.plhi.mhbs, PID: 4789
> 	java.lang.RuntimeException: Unable to create application edu.iupui.soic.biohealth.plhi.mhbs.mhbsTrainingApplication: android.util.AndroidRuntimeException: Calling startActivity() from outside of an Activity  context requires the FLAG_ACTIVITY_NEW_TASK flag. Is this really what you want?
>         at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4710)
>         at android.app.ActivityThread.-wrap1(ActivityThread.java)
>         at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1405)
>         at android.os.Handler.dispatchMessage(Handler.java:102)
>         at android.os.Looper.loop(Looper.java:148)
>         at android.app.ActivityThread.main(ActivityThread.java:5417)
>         at java.lang.reflect.Method.invoke(Native Method)
>         at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
>         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
>     Caused by: android.util.AndroidRuntimeException: Calling startActivity() from outside of an Activity  context requires the FLAG_ACTIVITY_NEW_TASK flag. Is this really what you want?
>         at android.app.ContextImpl.startActivity(ContextImpl.java:672)
>         at android.app.ContextImpl.startActivity(ContextImpl.java:659)
>         at android.content.ContextWrapper.startActivity(ContextWrapper.java:331)
>         at edu.iupui.soic.biohealth.plhi.mhbs.mhbsTrainingApplication.onCreate(mhbsTrainingApplication.java:46)
>         at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1013)
>         at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4707)
>         at android.app.ActivityThread.-wrap1(ActivityThread.java) 
>         at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1405) 
>         at android.os.Handler.dispatchMessage(Handler.java:102) 
>         at android.os.Looper.loop(Looper.java:148) 
>         at android.app.ActivityThread.main(ActivityThread.java:5417) 
>         at java.lang.reflect.Method.invoke(Native Method) 
>         at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
>         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)

File: mHBS_training/app/src/main/java/edu/iupui/soic/biohealth/plhi/mhbs/activities/SharedLoginActivity.java [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+        finish();
+        System.exit(0);
+        super.onBackPressed();
+    }
+}
+
+
+
+
+/*
```
Comment (sunbiz) [inline mHBS_training/app/src/main/java/edu/iupui/soic/biohealth/plhi/mhbs/activities/SharedLoginActivity.java]: Please delete commented code

File: mHBS_training/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -3,60 +3,58 @@
...
+            android:label="@string/title_activity_main"
+            android:theme="@style/AppTheme.NoActionBar">
+        </activity>
+
         <activity android:name=".activities.ProgramPortalActivity" />
         <activity android:name=".activities.ResourcesActivity" />
         <activity
             android:name=".activities.VideosActivity"
-            android:label="@string/title_activity_videos" />
+            android:label="@string/title_activity_videos"
```
Comment (sunbiz) [inline mHBS_training/app/src/main/AndroidManifest.xml]: Please fix style to end tag on same line

File: .gitmodules [hunk 3]
```diff
@@ -2,3 +2,4 @@
 	path = mHBS_training/sdk
 	url = https://github.com/dhis2/dhis2-android-sdk.git
 	branch = tracker-capture
+	ignore = dirty
```
Comment (sunbiz) [inline .gitmodules]: I hope this is deliberate to avoid making change by mistake to the SDK submodule?

Comment (Ezmeyers) [reply .gitmodules]: Yes, that was the reason for adding it

**Pull request closed without merging**